packages/moonstone/ProgressBar/ProgressBarTooltip.js

```diff
@@ -142,5 +176,10 @@ const ProgressBarTooltipBase = kind({
 	}
 });
 
+ProgressBarTooltipBase.defaultSlot = 'tooltip';
```

Comment:
It works in Slider but not in ProgressBar (yet) because it lacks `Slottable` support. I'm adding that support in #1573 which standardizes tooltip-related props across all three components.